test/create_parameters.jl

```diff
@@ -0,0 +1,86 @@
+include("../src/Kinematic1D.jl")
+
+import CLIMAParameters as CP
+import CloudMicrophysics as CM
+import Thermodynamics as TD
+
+#! format: off
+function create_parameter_set(
+    out_dir::String,
+    toml_dict::CP.AbstractTOMLDict,
+    FTD = CP.float_type(toml_dict),
+    w1 = 2.0,
+    t1 = 600.0,
+)
+    FT = CP.float_type(toml_dict)
+    override_file = joinpath(out_dir, "override_dict.toml")
+    open(override_file, "w") do io
+        println(io, "[mean_sea_level_pressure]")
+        println(io, "alias = \"MSLP\"")
+        println(io, "value = 100000.0")
+        println(io, "type = \"float\"")
+        println(io, "[gravitational_acceleration]")
+        println(io, "alias = \"grav\"")
+        println(io, "value = 9.80665")
+        println(io, "type = \"float\"")
+        println(io, "[gas_constant]")
+        println(io, "alias = \"gas_constant\"")
+        println(io, "value = 8.314462618")
+        println(io, "type = \"float\"")
+        println(io, "[adiabatic_exponent_dry_air]")
+        println(io, "alias = \"kappa_d\"")
+        println(io, "value = 0.2855747338575384")
+        println(io, "type = \"float\"")
+        println(io, "[isobaric_specific_heat_vapor]")
+        println(io, "alias = \"cp_v\"")
+        println(io, "value = 1850.0")
+        println(io, "type = \"float\"")
+        println(io, "[molar_mass_dry_air]")
+        println(io, "alias = \"molmass_dryair\"")
+        println(io, "value = 0.02896998")
+        println(io, "type = \"float\"")
+        println(io, "[molar_mass_water]")
+        println(io, "alias = \"molmass_water\"")
+        println(io, "value = 0.018015")
+        println(io, "type = \"float\"")
+        println(io, "[cloud_liquid_water_specific_humidity_autoconversion_threshold]")
+        println(io, "alias = \"q_liq_threshold\"")
+        println(io, "value = 0.0001")
+        println(io, "type = \"float\"")
+        println(io, "[prescribed_flow_w1]")
+        println(io, "alias = \"w1\"")
+        println(io, "value = " * string(w1))
+        println(io, "type = \"float\"")
+        println(io, "[prescribed_flow_t1]")
+        println(io, "alias = \"t1\"")
+        println(io, "value = " * string(t1))
+        println(io, "type = \"float\"")
+    end
+    toml_dict = CP.create_toml_dict(FT; override_file, dict_type="alias")
+    isfile(override_file) && rm(override_file; force=true)
+
+    aliases = string.(fieldnames(TD.Parameters.ThermodynamicsParameters))
+    aliases = (aliases..., "cp_d")
```

Comment:
cp_d is derived from other parameters and is not available in any toml file (neither default nor here); I don't understand why you extend aliases here by cp_d. I think get_parameter_values! doesn't return any value for cp_d. Maybe you can just remove this line.

Reply:
Right, good catch. At first I tried to provide a value for `cp_d`, but then realised its derived. So in the end I overwrite the `kappa_d` value, to get the `cp_d` same as in PySDM. 

I forgot to delete and will do so now.